documentation/modules/managing/proc-setting-broker-limits.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// assembly-management-tasks.adoc
+
+[id='proc-setting-broker-limits-{context}']
+
+= Setting limits on brokers using the Kafka Static Quota plugin
+
+[role="_abstract"]
+Use the _Kafka Static Quota_ plugin to set throughput and storage limits on brokers in your Kafka cluster.
+You can set a byte-rate threshold and storage quotas for clients interacting with your brokers.
```

Comment:
Does the byte-rate threshold apply to the clients or the brokers?

Reply:
I had to read line 11 a couple of times. "For clients" implies that the threshold and quotas are set in the clients rather than in the broker configuration.

Reply:
I changed the order and reworded:

Use the _Kafka Static Quota_ plugin to set throughput and storage limits on brokers in your Kafka cluster.
You enable the plugin and set limits by configuring the `Kafka` resource.
You can set a byte-rate threshold and storage quotas on clients interacting with your brokers.